application.yml

```diff
@@ -2,16 +2,18 @@ server:
   request_timeout: 4000ms
   shutdown_timeout: 4000ms
   port: 8080
+  max_body_bytes: 4000
+  max_header_bytes: 1000
 log:
   level: debug
   format: text
 storage:
   uri: postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
-# storage:
+#storage:
 #  name: sm-postgres
 api:
-  token_issuer_url: https://uaa.dev.cfdev.sh
-  client_id: sm
+  token_issuer_url: https://uaa.cf.sap.hana.ondemand.com
```

Comment:
Will **token_issuer_url** and **client_id** remain like so or is this just your local configuration?